ch03_Digikoppeling Restful API profiel.md

```diff
@@ -1,42 +1,95 @@
+
+* Zie [FSC](https://commonground.gitlab.io/standards/fsc/)
+
 #### Vertrouwelijkheid
 
-De Digikoppeling Beveiligingsstandaarden en voorschriften gaan specifiek in op het verplichte gebruik van PKIO certificaten [[PKI-Policy]] en 2-zijdig TLS.
+De Digikoppeling Beveiligingsstandaarden en voorschriften gaan specifiek in op het verplichte gebruik van PKIO certificaten [[PKI-Policy]] .
 * Zie [[Digikoppeling-Beveiligingsdocument]]
 
+FSC spreekt over een Thrust Anchor die door een Group moet worden gekozen. De Trust Anchor is binnen de context van X.509 certificaten de certificate authority (CA) waaruit het vertrouwen wordt afgeleid.
+
+De Trust Anchor voor de FSC Group moet daarom de PKIO Private Root zijn.
 
 #### Identificatie & Authenticatie
 
-Digikoppeling maakt gebruik van het OIN (Organisatie Identificatie Nummer) voor de identificatie van organisaties.
-Binnen dit DK REST-API profielprofiel zijn er alleen voorschriften m.b.t. het verplicht gebruik van het OIN binnen PKIO certificaten. Voor OIN gebruik binnen payloads (bv JSON) of resource-pad gelden geen specifieke voorschriften.
+Digikoppeling maakt gebruik van het OIN (Organisatie Identificatie Nummer) voor de identificatie van organisaties. Dit is het PeerID binnen de context van FSC. Het OIN wordt bij PKIO certificaten geplaatst in het SerialNumber veld van het Subject.
+Het verplicht vanuit FSC om te bepalen welk veld uit het certificaat de Peer name bepaald. Dit is het organization veld van het Subject van het PKIO certificaat.
+Binnen dit Digikoppeling REST API profielprofiel zijn er alleen voorschriften m.b.t. het verplicht gebruik van het OIN binnen PKIO certificaten en FSC. Voor OIN gebruik binnen payloads (bv JSON) of resource-pad gelden geen specifieke voorschriften.
 * Zie [[Digikoppeling-Identificatie-Authenticatie]]
 
+#### TLS 
+
+De Digikoppeling Beveilingsstandaarden en voorschriften veplichten het gebruik van 2-zijdig TLS met minimaal TLS versie 1.2, FSC verscherpt deze eis door de ciphersuites die geen perfect forward secrecy ondersteunen niet toe te laten.
+* Zie [[Digikoppeling-Beveiligingsdocument]]
+
+#### Netwerk-poorten
+
+De Digikoppeling Beveiligingsstandaarden en voorschriften verplichten het gebruik van de netwerkpoort 443 voor data verkeer. FSC voegt daar het gebruik van port 8443 voor managementverkeer aan toe. E.g. toegang aanvragen voor een API. 
+* Zie [[Digikoppeling-Beveiligingsdocument]]
+
+#### Contracten
+
+FSC gebruikt Contracten om afspraken tussen Peers vast te leggen. Een Contract kan één of meerdere Grants bevatten. Een Grant beschrijft welke interactie er mogelijk is tussen de Peers.
+FSC plaatst geen beperking op het aantal Grants per Contract. Het Digikoppeling REST API profiel doet dit wel om te voorkomen dat er fragiele Contracten ontstaan met een hoge beheerslast. Het aantal Grants wordt beperkt tot maximaal 10.
```

Comment:
Deze beperking van 10 lijkt mij in praktijk wel een issue. Hoe gaan we hier mee om? Meerdere contracten opstellen? De beperking van 10 kon ik ook niet zo snel vinden in het REST API profiel (link toevoegen?)